eras/conway/impl/src/Cardano/Ledger/Conway.hs

```diff
@@ -26,7 +26,8 @@ import Cardano.Ledger.Conway.Rules ()
 import Cardano.Ledger.Conway.Transition ()
 import Cardano.Ledger.Conway.Translation ()
 import Cardano.Ledger.Conway.Tx ()
-import Cardano.Ledger.Conway.TxInfo ()
+
+-- import Cardano.Ledger.Conway.TxInfo ()
```

Comment:
This import is necessary. It brings orphan instances into scope toegther with the ConwayEra type in order to un-orphan them.
```suggestion
import Cardano.Ledger.Conway.TxInfo ()
```